### PR TITLE
Add AWS Lambda Integration

### PR DESCRIPTION
**Pull request by agardnerIT**
Adds instructions for:

- Triggering Keptn sequences from AWS Lamba
- Triggering AWS Lambda functions from Keptn (and sending `.finished` events back to Keptn)

Signed-off-by: adamgardnerit <adam@agardner.net>

### CODE REVIEW

File: aws_lambda/artifacthub-pkg.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  - aws
+links:
+  - name: Source
+    url: https://aws.amazon.com/lambda
+recommendations:
+  - url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: "webhook"
+  keptn/version: "0.10.0-0.15.0"
+install: |
```
Comment (TannerGilbert) [inline aws_lambda/artifacthub-pkg.yml:23]: I recommend moving away from reusing the text from the README inside the installation section. I don't think it adds any additional value and makes the integration harder to maintain.

**Review by TannerGilbert — CHANGES_REQUESTED** [review]

File: aws_lambda/artifacthub-pkg.yml (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply aws_lambda/artifacthub-pkg.yml:23]: Fully agree. Please only use `install` section for actual installation instructions (e.g., `helm install ...`  for kubernetes based integrations). For webhooks, it's enough to go through the README.

Comment (agardnerIT) [reply aws_lambda/artifacthub-pkg.yml:23]: Makes sense. I'm sure we had a conversation in the past about that it was good to paste from readme into install section, but I can't find the discussion. Regardless, less duplication is good in my books. I'll remove the install section from all the integrations I manage (and open PRs) to fix up. So don't merge this until it's all ready.

**Comment by agardnerIT** [comment]
@TannerGilbert should be good now

**Review by TannerGilbert — APPROVED** [review]
LGTM. Thank you for updating the yml file.

**Pull request merged**